x-pack/plugins/cross_cluster_replication/public/__jest__/client_integration/follower_indices_list.test.js

```diff
@@ -334,7 +333,7 @@ describe('<FollowerIndicesList />', () => {
       test('should have a "settings" section', async () => {
         await actions.clickFollowerIndexAt(0);
         expect(find('followerIndexDetail.settingsSection').find('h3').text()).toEqual('Settings');
-        expect(exists('followerIndexDetail.settingsValues')).toBe(true);
+        expect(find('followerIndexDetail.settingsValues').length).toBe(10);
```

Comment:
Drive-by comment 😅 - are we guaranteed to always have `10` (e.g., is this something we're setting as part of the test setup), or if there the possibility for the value to change in the future?

Reply:
Heya, no worries all comments are welcome! Unless the follower index is paused (which it isnt for this particular test  suite) its garanteed to always have 10 setting values shown. I agree perhaps 10 isnt too future proof, what do you think about checking if length is > 0 instead? 🤔

Reply:
I think checking length is > 0 is probably more future-proof 👍  I know when we've specified numbers like this in the past, it has sometimes caused issues.

Reply:
Sounds good! I've created https://github.com/elastic/kibana/pull/150046 to address this